projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -16,13 +16,46 @@ class Jetpack_Podcast_Helper {
 	 */
 	protected $feed = null;
 
+	/**
+	 * The number of seconds to cache the podcast feed data.
+	 * This value defaults to null, so we don't override the core filters for RSS feeds in WordPress.
+	 * The value can be overridden specifically for podcasts using the
+	 * `jetpack_podcast_feed_cache_timeout` filter. Note that the cache timeout value
+	 * for all RSS feeds can be modified using the `wp_feed_cache_transient_lifetime`
+	 * filter from WordPress core.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/wp_feed_cache_transient_lifetime/
+	 * @see WP_Feed_Cache_Transient
+	 *
+	 * @var int|null
+	 */
+	protected $cache_timeout = null;
+
 	/**
 	 * Initialize class.
 	 *
 	 * @param string $feed The RSS feed of the podcast.
 	 */
 	public function __construct( $feed ) {
 		$this->feed = esc_url_raw( $feed );
+
+		/**
+		 * Filter the number of seconds to cache a specific podcast URL for. The returned value will be ignored if it is null or not a valid integer.
+		 * Note that this timeout will only work if the site is using the default `WP_Feed_Cache_Transient` cache implementation for RSS feeds,
+		 * or their cache implementation relies on the `wp_feed_cache_transient_lifetime` filter.
+		 *
+		 * @since $$next-version$$
+		 * @see https://developer.wordpress.org/reference/hooks/wp_feed_cache_transient_lifetime/
+		 *
+		 * @param int|null $cache_timeout The number of seconds to cache the podcast data. Default value is null, so we don't override any defaults from existing filters.
+		 * @param string   $podcast_url   The URL of the podcast feed.
+		 */
+		$podcast_cache_timeout = apply_filters( 'jetpack_podcast_feed_cache_timeout', $this->cache_timeout, $this->feed );
+
+		// Make sure we force new values for $this->cache_timeout to be integers.
+		if ( is_int( $podcast_cache_timeout ) ) {
```

Comment:
What if someone were to pass a string to the filter:

```php
add_filter( 'jetpack_podcast_feed_cache_timeout', function () { return '42'; } );
```

Maybe we should take that into account, and cast everything as an int as long as it's not null?

```suggestion
		$podcast_cache_timeout = apply_filters( 'jetpack_podcast_feed_cache_timeout', $this->cache_timeout, $this->feed );

		// Make sure we force new values for $this->cache_timeout to be integers.
		if ( $podcast_cache_timeout !== null ) {
```

Reply:
@jeherve, would it make more sense to keep the logic as-is but check for `is_numeric()` instead? I don't mind keeping the logic looser and simply casting to an int for any non-null value, but I am happy to defer to the Jetpack team's preference on this one!

Reply:
Also, thanks for the ultra-quick review!

Reply:
`is_numeric()` would work, sure thing! That works for me.

Reply:
I've switched to `is_numeric()` in 6138202.